### PR TITLE
MNT bump version on main

### PR DESCRIPTION
**Pull request by adrinjalali**
cc @skops-dev/maintainers

### CODE REVIEW

**Comment by BenjaminBossan** [comment]
For my understanding, why `0.3.dev0` and skipping `0.2.0`?

**Comment by adrinjalali** [comment]
Because we forgot to do this last time. 0.2 is the version for the branch which is already up under `0.2.X` now

**Comment by BenjaminBossan** [comment]
So if we do a release now, what version would it package have on pypi?

**Comment by adrinjalali** [comment]
0.2

File: skops/__init__.py [hunk 1]
```diff
@@ -16,7 +16,7 @@
 # Dev branch marker is: 'X.Y.dev' or 'X.Y.devN' where N is an integer.
 # 'X.Y.dev0' is the canonical version of 'X.Y.dev'
 #
-__version__ = "0.1.dev0"
+__version__ = "0.3.dev0"
```
Comment (merveenoyan) [inline skops/__init__.py:19]: LGTM!

**Review by merveenoyan — APPROVED** [review]
Thanks for the PR.

**Comment by BenjaminBossan** [comment]
What's up with codecov? It shows [these lines](https://app.codecov.io/gh/skops-dev/skops/compare/115/changes) to have uncovered changes, even though they were not added in this PR :-/

Also, CI is really strange, doc tests go from 150 sec to 60 min timeouts.

**Comment by adrinjalali** [comment]
I don't think codecov is related here anyway. It does go flaky sometimes in my experience.

We should merge this one since the release is already out.

**Pull request merged**